components/mission_key_management/client.c

```diff
@@ -101,14 +203,24 @@ size_t client_buffer_size(struct client* c) {
         case CS_DONE:
             return 0;
 
-        default:
-            // Unreachable
+        default: { 
+            /*$ assert(false); $*/  // <-- Prove this state is unreachable
             return 0;
+        } 
     }
 }
 
 
-int client_epoll_ctl(struct client* c, int epfd, int op) {
+int client_epoll_ctl(struct client* c, int epfd, int op) 
+/*$
+// TODO fill in an actual spec here, depending what's needed 
```

Comment:
@septract please either provide specs, or explain why we are not providing them (e.g. unsupported by the CN)